src/ScoreClosure.fs

```diff
@@ -53,35 +53,58 @@ let rec toScoreClosure (V: SymbolicValue) =
     | None ->
         // The function could not be translated to a linear function directly. Match on the top operation
         match V with
-        | SVFun (PDF_NORMAL, [| SVCon mean; SVCon sigma; W |]) | SVFun (PDF_NORMAL, [| W; SVCon sigma; SVCon mean |]) ->
-            // For the normal function, we assume that either the mean or the argument is constant.
-            // We then convert the argument to a score closure.
-            // Given bounds on the argument, BoundNonLinearPart, the computes lower and upper approximations on the value of the normal pdf on that interval
+        | SVFun (PDF_NORMAL, [| SVCon mean; sigma; W |]) | SVFun (PDF_NORMAL, [| W; sigma; SVCon mean |]) -> 
+            // We assume that at least one the mean or the argument is a constant. The remaning arguments may be arbitrary symbolic values
 
-            let recRes = toScoreClosure W
+            // Convert the two symbolic values to score closures
+            let scoreClosureSigma = toScoreClosure sigma 
+            let scoreClosureW = toScoreClosure W
 
-            // The lower bound on the pdf normal is taken at one of the ends of the interval
-            let normLowerApprox =
-                fun x y -> min (pdfNormal (mean, sigma, x)) (pdfNormal (mean, sigma, y))
+            let lowerBound (sigma_l, sigma_u) (w_l, w_u) =
+                // The smallest value is attained at the position in the w-interval that is furthest aways from the mean
+                let pointFurthestToMean = if abs(w_l - mean) > abs(w_u - mean) then w_l else w_u
 
-            // The upper bound on the pdf normal is taken at either one of the ends of the interval, or at the mean (if included in the interval)
-            let normUpperApprox x y =
-                if x <= mean && mean <= y then
-                    pdfNormal (mean, sigma, mean)
+                // The function \sigma -> pdfnormal(mean, sigma, pointClosedToMean) is monotone increasing on (0, t) and monotine decreasing on (t, \infty) for some value t (which we can compute but do not need here)

```

Comment:
ClosedTo -> ClosestTo (in other places too)
monotone -> monotonically (or just leave it out)
monotine -> monotonically (or just leave it out)